src/App.tsx

```diff
@@ -376,7 +390,7 @@ function App() {
         if (response.data[dataElementURI].length === 0) {
           // TODO: make into a toast
         } else {
-          setState(response.data[dataElementURI].sort((a, b) => a.Label.localeCompare(b.Label)));
+          setState(response.data[dataElementURI]);
```

Comment:
We should rename this, setState is an actual react method for class components: https://react.dev/reference/react/Component#setstate

Reply:
more generally, I think this could be cleaned in refactor by just returning `response.data` to where this function is called, and then use the right setter there.